bin/hi/main.C

```diff
@@ -664,6 +665,8 @@ void initHI(evaluator* eval, bool useDefColors) {
 
 int main(int argc, char** argv) {
   try {
+    hobbes::orc::Jit::init();
```

Comment:
Why wouldn't `hobbes::cc` take care of this?

Reply:
I'll definitely change this; this PR is giving a sense of where we're going with LLVM support in Hobbes, so as to gather early feedbacks.